### PR TITLE
fix(NcListItem): use font-size-small for subname and details

### PR DESCRIPTION
**Pull request by marcoambrosini**
### ☑️ Resolves

- Fix #…

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
 Screenshot 2024-08-05 at 17 36 52 | Screenshot 2024-08-05 at 17 36 28

### CODE REVIEW

File: src/components/NcListItem/NcListItem.vue [hunk 1]
```diff
@@ -1012,6 +1013,7 @@ export default {
 
 	&-details {
 		&__details {
+			font-size: var(--font-size-small);
 			color: var(--color-text-maxcontrast);
 			margin: 0 9px !important;
 			font-weight: normal;
```
Comment (susnux) [inline src/components/NcListItem/NcListItem.vue:1019]: Can we group the font related rules?
```suggestion
			color: var(--color-text-maxcontrast);
			font-size: var(--font-size-small);
			font-weight: normal;
			margin: 0 9px !important;
```

**Comment by ShGKme** [comment]
I'm not sure about this. `NcListItem`s are not limited in height like the top bar.

Could you also add examples with Talk and Mail to the PR?

**Comment by marcoambrosini** [comment]
Mail will have its own component @ShGKme. Talk will look kinda like this 
Screenshot 2024-08-05 at 18 23 45

**Comment by susnux** [comment]
> I'm not sure about this. `NcListItem`s are not limited in height like the top bar.

I think it looks nice to have the subline smaller, but I also think we should make this consistent with similar components like ActionLink

**Review by jancborchardt — COMMENTED** [review]
I also like this, gives it some nice contrast.

Also very related is the style we had a while ago where the main line of e.g. Talk conversations or Mails were bold. Maybe bold is a bit extreme there, but we could go for semibold.

File: src/components/NcListItem/NcListItem.vue [hunk 2]
```diff
@@ -877,6 +877,7 @@ export default {
 }
 
 .list-item-content__subname {
+	font-size: var(--font-size-small);
```
Comment (susnux) [inline src/components/NcListItem/NcListItem.vue:880]:
```suggestion
	font-size: var(--font-size-small, 13px);
```

File: src/components/NcListItem/NcListItem.vue [hunk 3]
```diff
@@ -1013,8 +1014,9 @@ export default {
 	&-details {
 		&__details {
 			color: var(--color-text-maxcontrast);
-			margin: 0 9px !important;
+			font-size: var(--font-size-small);
```
Comment (susnux) [inline src/components/NcListItem/NcListItem.vue:1017]:
```suggestion
			font-size: var(--font-size-small, 13px);
```

**Review by susnux — COMMENTED** [review]
I think this makes sense and I like the look. Just the fallback values are missing.

**Comment by marcoambrosini** [comment]
outdated

**Pull request closed without merging**